qiskit/circuit/library/data_preparation/pauli_feature_map.py

```diff
@@ -277,7 +279,7 @@ def zz_feature_map(
 
     Examples:
 
-        >>> from qiskit.circuit.library import ZZFeatureMap
+        >>> from qiskit.circuit.library.data_preparation import zz_feature_map
```

Comment:
We encourage high-level imports (the ones that are actually API documented) so it would be better to leave this as:
```suggestion
        >>> from qiskit.circuit.library import zz_feature_map
```